src/SourceBuild/content/test/Microsoft.DotNet.UnifiedBuild.Tests/NuGet.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="dotnet9" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9/nuget/v3/index.json" />
```

Comment:
Please add this feed to the VMR's root NuGet.config instead. We don't want test projects to need separate package sources.

Reply:
If setting the DotNetHostPath makes it so that we don't need to publish self-contained, then we actually shouldn't need this feed. I'll just remove this.

Reply:
Even better